.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: babel-loader
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint:
+    name: lint
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [12.x]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: yarn
+      - name: Lint
+        run: yarn run lint
+  test:
+    name: Test - ${{ matrix.os }} - Node v${{ matrix.node-version }}, Webpack ${{ matrix.webpack-version }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 12.x, 14.x]
+        webpack-version: [latest, next]
```

```suggestion
        webpack-version: [4]
```
Can you separate `"next"` into another PR? So we can merge this one without failing CI (because we haven't supported webpack@next yet). Also revised `"latest"` to `4` so it is future-proof.